src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -282,6 +282,37 @@ private void testEditQuestionAction() throws Exception {
                                 .contains("The recipient's team members can see your response, but not the name of the recipient, or your name."));
         
         feedbackEditPage.clickAndCancel(feedbackEditPage.getCancelQuestionLink(-1));
+        
+        ______TS("add question 2 and edit it to giver's team members and giver");
+        feedbackEditPage.clickAddQuestionButton();
+        feedbackEditPage.fillQuestionBox("test visibility when choosing giver's team members and giver");
+        feedbackEditPage.selectGiverToBeStudents();
+        feedbackEditPage.selectRecipientsToBeGiverTeamMembersAndGiver();
+        feedbackEditPage.clickMaxNumberOfRecipientsButton();
+        feedbackEditPage.clickAddQuestionButton();
+        
+        ______TS("test visibility options of question 2");
+        feedbackEditPage.clickVisibilityOptionsForQuestion(2);
+     
+        assertEquals(feedbackEditPage.getVisibilityOptionTableRow(2, 2).getText(), "Recipient(s)");
+        assertEquals(feedbackEditPage.getVisibilityOptionTableRow(2, 3).getText(), "Giver's Team Members");
+        assertEquals(feedbackEditPage.getVisibilityOptionTableRow(2, 4).getText(), "Other students");
+        assertEquals(feedbackEditPage.getVisibilityOptionTableRow(2, 5).getText(), "Instructors");
```

Comment:
@Sumei1009, swap the order of the parameters, it's <code>assertEquals(expected, actual)</code>.